adviser/overlays/ocp4-stage/kustomization.yaml

```diff
@@ -91,3 +91,9 @@ patches:
       version: v1alpha1
       kind: WorkflowTemplate
       name: dm
+  - path: adviser-rolebinding-patch.yaml
+    target:
+      group: apps
+      version: v1
+      kind: RoleBinding
+      name: adviser
```

Comment:
can you share the rbac of this adviser ?

Reply:
https://github.com/thoth-station/thoth-application/pull/2607

Will be changed to `send-webhook`